django/db/migrations/operations/fields.py

```diff
@@ -141,7 +141,7 @@ def state_forwards(self, app_label, state):
                 old_field = instance
         state.models[app_label, self.model_name_lower].fields = new_fields
         # Delay rendering of relationships if it's not a relational field
-        delay = not old_field.is_relation
+        delay = not old_field.is_relation if old_field else False
```

Comment:
Under which circumstances would `old_field` be `None` here? Could you write a test for that?

Reply:
I could find only case if it tries to remove an old_field which does not exist in the model anymore. 
Test added.